README.md

```diff
@@ -126,41 +123,108 @@ helm delete my-galaxy
+| `fullnameOverride`                        | Override the full name used to prefix resource names. Defaults to {{.Release.Name}}-{{.Values.nameOverride}} |
+| `image.repository`                        | The repository and name of the Docker image for Galaxy, searches Docker Hub by default |
+| `image.tag`                               | Galaxy Docker image tag (generally corresponds to the desired Galaxy version) |
+| `image.pullPolicy`                        | Galaxy image [pull policy](https://kubernetes.io/docs/concepts/configuration/overview/#container-images) for more info |
+| `imagePullSecrets`                        | Secrets used to [access a Galaxy image from a private repository](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) |
+| `service.type`                            | Kubernetes [Service type](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) |
+| `service.port`                            | Kubernetes service port                                      |
+| `service.nodePort`                        | If `service.type` is set to `NodePort`, then this can be used to set the port at which Galaxy will be available on all nodes' IP addresses |
+| `workflowHandlers.*`                      | Configuration for the workflowHandlers (See below for all options) |
+| `webHandlers.*`                           | Configuration for the webHandlers                            |
+| `jobHandlers.*`                           | Configuration fo the jobHandlers                             |
+| `jobHandlers.priorityClass.enabled`       | Assign a [priorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass) to this handler. |
+| `jobHandlers.priorityClass.existingClass` | The [priorityClass](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass) to assign. |
+| `metrics.enabled`                         | Enable metrics gathering.                                    |
+| `metrics.image.repository`                | The location of the [galay-metrics-scraping](https://github.com/CloudVE/galaxy-docker-k8s-metrics) image to use. |
+| `metrics.image.tag`                       | The image version to use.                                    |
+| `metrics.image.pullPolicy`                | Define the [pull policy](https://kubernetes.io/docs/concepts/containers/images/#updating-images), that is, when Kubernetes will pull the image. |
+| `serviceAccount.create`                   | The serviceAccount will be created if it does not exist.     |
+| `serviceAccount.name`                     | The serviceAccount account to use.                           |
+| `rbac.enabled`                            | Enable Galaxy job RBAC                                       |
```

Comment:
```suggestion
| `rbac.enabled`                            | Enable Galaxy job RBAC. This will grant the service account the necessary permissions/roles to view jobs and pods in this namespace. Defaults to true.  |
```